network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -295,6 +310,45 @@ class ProductPage(FoundationMetadataPageMixin, Page):
         blank=True,
     )
 
+    # Un-editable voting fields. Don't add these to the content_panels.
+    current_total = models.IntegerField(default=0)
```

Comment:
Hm, what is the benefit of having an explicit `current_total` field if that value is implicit to the sum of `vote_bins`?

Reply:
One less database query per product. Though this is only for the ProductPage and not the Home/listing page, so one more db query would be OK imo. Ill update this with the new current_total being the sum of al 5 bins, once I change the bin_1, bin_2, etc to a CharField with `int_list_validator`